spec/haskell/src/SEL4/Object/TCB.lhs

```diff
@@ -1014,12 +1011,10 @@ On some architectures, the thread context may include registers that may be modi
 
 > chargeBudget :: Ticks -> Bool -> Kernel ()
 > chargeBudget consumed canTimeoutFault = do
->     stateAssert cur_tcb'_asrt
->         "Assert that `cur_tcb' s` holds"
->     stateAssert sch_act_wf_asrt
->         "Assert that `sch_act_wf (ksSchedulerAction s) s` holds"
+>     stateAssert cur_tcb'_asrt "`cur_tcb'`"
+>     stateAssert sch_act_wf_asrt "`sch_act_wf (ksSchedulerAction s) s`"
```

Comment:
I'm wondering if we get much benefit from the backticks, considering they're a concept for markdown. I'm fine leaving them in, but thought I'd raise it. @lsf37 ?

Reply:
Yeah I wasn’t sure if we make some sort of document out of the Haskell. I’m used to using them on Mattermost and I kind of like how it separates terms we use within Isabelle from regular words

Reply:
I guess we can adapt the practice. We already use it a bit in Isabelle `(* *)` comments for example, so that words don't get confused with terms (since `@{term }` is so long-winded and more for text<> and \<comment> comments), but we're very irregular about it. It's more when there's nothing to differentiate, like here, that it looks odd.